docs/users.rst

```diff
@@ -26,20 +41,158 @@ image owners can use ``koji watch-logs <task id> | grep -w x86_64``.
 Configuring koji-containerbuild for koji CLI
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+To be able to build container images from CLI `koji-containerbuild plugin`_
+must be installed.
+
+No additional configuration is required everything should be set by ``koji`` package.
+
+.. _`koji-containerbuild plugin`: https://github.com/containerbuildsystem/koji-containerbuild
+
+
+Koji Build Results
+~~~~~~~~~~~~~~~~~~
+
+Koji Web
+********
+
+This is the easiest way to access information about OSBS builds.
+
+List Builds
++++++++++++
+
+Navigate to the “Builds” tab in brew and set the “Type” filter to ``image``.
```

Comment:
Any reason for using these special quotes chars? They are spread in other parts of the patch as well. It would be nice to substitute them for regular quotes (if they are not here for any special reason) to make parsing  the docs easier.